src/task-mark-module.js

```diff
@@ -0,0 +1,42 @@
+export const taskMarking = () => {
+  const taskCheck = document.querySelectorAll('.task-check');
+  taskCheck.forEach((chk, index) => {
+    chk.addEventListener('click', () => {
+      if (chk.checked === true) {
+        document.getElementById(`label_${index}`).style.textDecoration = 'line-through';
+        document.getElementById(`label_${index}`).style.color = '#a9a9a9';
+
+        const chkTaskListSet = JSON.parse(JSON.stringify(localStorage.getItem('taskList')));
+        const newChkTaskListObjSet = JSON.parse(chkTaskListSet);
+        newChkTaskListObjSet[index].completed = true;
+        localStorage.setItem('taskList', JSON.stringify(newChkTaskListObjSet));
+      } else {
+        document.getElementById(`label_${index}`).style.textDecoration = 'none';
+        document.getElementById(`label_${index}`).style.color = '#000';
+
+        const chkTaskListSet = JSON.parse(JSON.stringify(localStorage.getItem('taskList')));
+        const newChkTaskListObjSet = JSON.parse(chkTaskListSet);
+        newChkTaskListObjSet[index].completed = false;
+        localStorage.setItem('taskList', JSON.stringify(newChkTaskListObjSet));
+      }
+    });
+  });
+};
+
+export const clearCompletedtask = (clearbtn, resetObjIndexFunction, loadObjFunction) => {
+  clearbtn.addEventListener('click', () => {
+    const clearTaskListSet = JSON.parse(JSON.stringify(localStorage.getItem('taskList')));
+    const newClearTaskListObjSet = JSON.parse(clearTaskListSet);
+    const newArray = [];
+
+    for (let i = 0; i < newClearTaskListObjSet.length; i += 1) {
+      if (newClearTaskListObjSet[i].completed === false) {
+        newArray.push(newClearTaskListObjSet[i]);
+      }
+    }
+
```

Comment:
- [ ] To clear all completed tasks you were required to use the (filter) method, `Array.filter()` :point_down: 

![image](https://user-images.githubusercontent.com/75126481/165750493-90b3cabf-1b44-4e27-aeb0-51f61eb7d971.png)